core/src/main/java/me/lokka30/treasury/api/common/event/EventExecutorTrackerShutdown.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * This file is/was part of Treasury. To read more information about Treasury such as its licensing, see <https://github.com/lokka30/Treasury>.
+ */
+
+package me.lokka30.treasury.api.common.event;
+
+public class EventExecutorTrackerShutdown {
```

Comment:
missing javadoc

Reply:
thats an implementation detail ; we don't need javadoc